JeanetteKnipp-Kursdoku

Lellahahaha opened the pull request:
Awesome you want to make a PR  
please see [https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md](https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md) for further infos.

fabianmoronzirfas commented on projects/DMA-Enrico-Castellani/index.md:
```diff
@@ -0,0 +1,83 @@
+type: project
+---
+
+![](./assets/castellani.png) 
```
No splash.png image? 
no subfolder called images?

fabianmoronzirfas commented on projects/DMA-Enrico-Castellani/index.md:
```diff
@@ -0,0 +1,83 @@
+---
+
+![](./assets/castellani.png) 
+# Deconstructing Master Artists
```
The Headline h1 (`#`) will be added by the system form your title. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax

Lellahahaha replied:
tried it, doesn't work for me.

fabianmoronzirfas commented on projects/DMA-Enrico-Castellani/index.md:
```diff
@@ -0,0 +1,83 @@
+
+------
+
+We tried to express the laws used by Castellani in code to create generative art. This is the result:
```
You are very short on your outcome…

Lellahahaha replied:
changed it.

fabianmoronzirfas commented on projects/this-is-not-a-leaf/index.md:
```diff
@@ -0,0 +1,125 @@
+
+The revised algorithm now looks like this:
+
+*Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.*
```
maybe better as quote:   

> Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.

done like this  

    > Draw ovals that thin to the tails. crisscross, in different sizes, almost touching. Fill some of them with lines or dots and leave others empty.

Lellahahaha replied:
done.

fabianmoronzirfas commented on projects/this-is-not-a-leaf/index.md:
```diff
@@ -0,0 +1,125 @@
+![](./assets/ergebnis12.png)
+![](./assets/ergebnis13.png)
+
+As you can see, with this new algorithm it still works out. All correct results! But a little corny now. So let’s stay with the pseudo-leafs and change the canvas! No regular sheets of paper allowed anymore. But instead - wrapping paper, an egg, even skin.
```
Please write from a bit more objective perspective. No slang, no "let's try…" Write as it where an analysis

Lellahahaha replied:
changed

fabianmoronzirfas commented on projects/this-is-not-a-leaf/index.md:
```diff
@@ -0,0 +1,125 @@
+![](./assets/teller1.png)
+![](./assets/teller2.png)
+
+Yassss!
```
please… This is not twitter or facebook.

fabianmoronzirfas commented:
@Lellahahaha are you done? can I review it again?

fabianmoronzirfas commented:
There is still the question of the image rights open.
Also you've added a folder on the root level of the project called images:  

https://github.com/Lellahahaha/gestalten-in-code/tree/e330320be8063031a5e5384945b57ca58a216a5e/images 

Please remove it. All your images should be in your project/sketches folders

fabianmoronzirfas commented on projects/DMA-Enrico-Castellani/index.md:
```diff
@@ -0,0 +1,89 @@
+
+![](./assets/images/castellanicode.png)
+
+In concliusion, even though Castellani's work is "only" higher and lower placed nails on an evenly colored surface it's very hard to imitate using code. The nails use light and give shadow to appear as dots, lines and whole forms in a three dimensional space which is, for p5.js-beginners, a great task to authentically simulate - only having the knowledge to use basic shapes like circles and conne ...
```
> concliusion
conclusion

fabianmoronzirfas commented:
So? Any development on the images?

Lellahahaha commented:
Almost.
Unfortunately, I couldn't find any license free fitting high resolution images so far (although I have been searching for them on at least three different Pages.)
Will try it again now - if I still can't find any I'll just remove the Images from the document. Will update it till 7PM today.

fabianmoronzirfas commented:
Hm. Sad. If you find anything low res use that

Lellahahaha commented:
Alright!

Lellahahaha commented:
Okay, so:

Changed the splash and thumb Image into one that I don't Need a license for. Also, I found one license free Picture as an example for Castellani's work. But except for that I really could not find any free Images. Neither of Castellani himself, nor of his work or the working process. So I removed all the Images I do not have a license for from the document and the Image Folder. But: what do I do about the examples Natalie and I made in Illustrator by only covering existing Images of castellani's work with light lines? Do I have to delete them as well because we don't have the license for the original Pictures? Or should I just blur them a Little? Or can I leave them as they are?

fabianmoronzirfas commented:
> Do I have to delete them as well because we don't have the license for the original Pictures? Or should I just blur them a Little? Or can I leave them as they are?

Let's keep it that way for now.

fabianmoronzirfas commented:
merged and live https://interface.fh-potsdam.de/gestalten-in-code/projects/DMA-Enrico-Castellani/ && https://interface.fh-potsdam.de/gestalten-in-code/projects/this-is-not-a-leaf/
